docs/components/App.js

```diff
@@ -28,6 +29,7 @@ export default function App({ children }: Props): Node {
       event.preventDefault();
       router.push(href);
     };
+    setIsHomePage(window.location.pathname === '/home');
```

Comment:
@dangerismycat What's the best way to do this? Window comes back as undefined in a flaky way, even if changing to window?.location?.pathname